src/engagement.py

```diff
@@ -0,0 +1,115 @@
+from core_data_modules.util import PhoneNumberUuidTable
+
+from src.lib.pipeline_configuration import PipelineConfiguration
+
+class Engagement(object):
+    @staticmethod
+    def generate(data, phone_number_uuid_table):
```

Comment:
```suggestion
    def compute_statistics(data, phone_number_uuid_table):
```
perhaps? (You're not actually generating engagement here :P)